doc/source/remote_reading.rst

```diff
@@ -0,0 +1,166 @@
+====================
+Reading remote files
+====================
+
+Using a single reader
+=====================
+
+Some of the readers in Satpy can read data directly over various transfer protocols. This is done
+using `fsspec <https://filesystem-spec.readthedocs.io/en/latest/index.html>`_ and various packages
+it is using underneath.
+
+As an example, reading ABI data from public AWS S3 storage can be done in the following way::
+
+    from satpy import Scene
+
+    storage_options = {'anon': True}
+    filenames = ['s3://noaa-goes16/ABI-L1b-RadC/2019/001/17/*_G16_s20190011702186*']
+    scn = Scene(reader='abi_l1b', filenames=filenames, reader_kwargs={'storage_options': storage_options})
+    scn.load(['true_color_raw'])
+
+Reading from S3 as above requires the `s3fs` library to be installed in addition to `fsspec`.
+
+As an alternative, the storage options can be given using
+`fsspec configuration <https://filesystem-spec.readthedocs.io/en/latest/features.html#configuration>`_.
+For the above example, the configuration could be saved to `s3.json` in the `fsspec` configuration directory
+(by default placed in `~/.config/fsspec/` directory in Linux)::
+
+    {
+        "s3": {
+            "anon": "true"
+        }
+    }
+
+.. note::
+
+    Using `reader_kwargs` doesn't completely override the configuration file, so in case of problems
+    remove the configuration file to see if that solves the issue.
```

Comment:
Does this mean the options specified in `reader_kwargs` only "supplement" or add on to the options specified in the config?

Reply:
Nah, because then you couldn't run it with the secret keys in the config and the anon access stuff in the kwargs. As-is is probably fine.

Reply:
Or...how does xarray do it?

Reply:
Oh yeah, having the secrets in a config is better than having them in code.

Reply:
Clarified the note a bit.